### PR TITLE
Draft summary 1 for smith2015cyclical

### PR DESCRIPTION
**Pull request by liaojh1998**
To be done by tonight.

### CODE REVIEW

**Comment by liaojh1998** [comment]
Draft is almost complete, but I'll add more tomorrow after class.

**Comment by liaojh1998** [comment]
Hi @trowk, the draft summary is completed so you can review it whenever.

**Review by trowk — APPROVED** [review]
You did an excellent job summarizing the paper! The main idea of the paper was succinct and informed the reader what a cyclical learning rate was and why the authors believe it works. The next section then quickly explains how this cyclical learning rate is implemented and why he chose a triangular function. You even included the recommendations the author made on how to choose hyperparameters for the cyclical learning rate function (base and max lr, step size, and the stopping point).

In the following section, you provide an excellent summary of the two methods Smith used to “decay” the cyclical learning rate. Finally, you quickly summarized the results of the author’s experiment and provided a picture of a table that showed the final accuracies for his experiments with adaptive learning rates.

My only suggestions would be to mention that the author proved that both increasing and decreasing the learning rate is essential to the improved performance. Smith noted that the accuracy increased the most when the learning rate was smaller, but went out of his way to prove that only decreasing the learning rate did not yield the same benefits as the cyclical learning rate function in an experiment (top of page 5). Furthermore, you might want to also provide a picture of the other set of experiments the other performed (table on top of page 6 that shows how well the different architectures performed). Finally, I know you mentioned that the cyclical learning rate function can harm performance in the short term, but I thought that the graph of “ImageNet with GoogleNet architecture” on page 8 provided a great illustration of this point and really provided the readers a concrete understanding of how this cyclical learning rate function can affect training (including the graph is just a suggestion. If you feel that you have enough pictures as it is, you can leave it out). Other than that,I thought the summary was excellent!

I thought I should also quickly mention that I was unable to view the first two images and the last image in markdown. I tried viewing the page both with firefox and chrome but the images would not appear. Looking at the html and directly accessing the link, however, did work, so it’s entirely possible that this is just a problem with my computer. I just thought I would mention it.

**Comment by liaojh1998** [comment]
@trowk Thank you so much for the review Ian! Here's a list of my changes:
* Corrected all datasets summary table (previously used the different adaptive methods table in the wrong place).
* Added different adaptive methods results table.
* Added GoogleNet training on ImageNet figure to show how CLR benefits overall.

I did not include `decay` experiment mentioned on the first paragraph of page 5 as I felt that the experiment is flawed. The `triangular2` learning rates were much smaller (dropping from 0.001 to 0.00001 in `base_lr` as shown in Table 2), whereas the `decay` experiment decreased linearly from 0.007 to 0.001 and kept the LR fixed at 0.001 for the entirety of the training after. Furthermore, the `fixed` policy for CIFAR-10 in Table 1 (with 70K iterations) performed the same as `triangular2` policy (with 25K iterations). Therefore, I only believed that CLR can speed up training to reach the same accuracy (under less iterations), and this is mentioned in #L82. If you think I should still include this experiment, let me know.

**Comment by liaojh1998** [comment]
For the image issue, it seems that the images cannot be displayed on GitHub on Markdown, but the images can be displayed on Jekyll. This is also the case for my previous summary.

I used `jekyll serve` for preview and it seems that the site is working on my side. If there are issues still let me know.

**Comment by liaojh1998** [comment]
Rebased for merge.
